openmetadata-ui/src/main/resources/ui/src/components/Explore/Explore.component.tsx

```diff
@@ -53,7 +58,14 @@ const Explore: React.FC<ExploreProps> = ({
   loading,
 }) => {
   const isMounting = useRef(true);
+  const { tab } = useParams() as Record<string, string>;
+
+  // get entity active tab by URL params
+  const defaultActiveTab = () => {
+    const entityName = tab.slice(0, -1).toUpperCase();
```

Comment:
Please do safe handling like tab.slice(0, -1) is a string and won't throw an exception